tools/cythonize.py

```diff
@@ -168,13 +168,13 @@ def process(path, fromfile, tofile, processor_function, hash_db):
     fulltopath = os.path.join(path, tofile)
     current_hash = get_hash(fullfrompath, fulltopath)
     if current_hash == hash_db.get(normpath(fullfrompath), None):
-        print('%s has not changed' % fullfrompath)
+        sys.stderr.write('%s has not changed' % fullfrompath)
```

Comment:
Note that `sys.stderr.write` does not include the trailing newline. May I suggest doing this instead?

```suggestion
        print('%s has not changed' % fullfrompath, file=sys.stderr)
```